pilot/cmd/pilot-agent/main.go

```diff
@@ -144,8 +144,8 @@ var (
 	useTokenForCSREnv   = env.RegisterBoolVar("USE_TOKEN_FOR_CSR", false, "CSR requires a token").Get()
 	credFetcherTypeEnv  = env.RegisterStringVar("CREDENTIAL_FETCHER_TYPE", "",
 		"The type of the credential fetcher. Currently supported types include GoogleComputeEngine").Get()
-	parseTokenEnv = env.RegisterBoolVar("PARSE_TOKEN", false,
-		"Parse token to inspect information like expiration time in proxy. This may not always be possible because token may not be a JWT.").Get()
+	skipParseTokenEnv = env.RegisterBoolVar("SKIP_PARSE_TOKEN", false,
+		"Skip Parse token to inspect information like expiration time in proxy. This may not always be possible because token may not be a JWT.").Get()
```

Comment:
"Skip parse token", could you also rephrase the flag description?

Reply:
updated it